client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -80,7 +78,7 @@ export const InboundReturnEditModal = ({
         }
         okButton={
           currentTab === Tabs.Reason ? (
-            <DialogButton onClick={onOk} variant="ok" disabled={!okEnabled} />
```

Comment:
we have reasons as optional in the backend, so have removed the constraint here 😅